### PR TITLE
Settingspage

### PR DESCRIPTION
**Pull request by Adebayor7**
Created a new branch where i worked on my settings.tsx page

### CODE REVIEW

**Review by pablo-clueless — CHANGES_REQUESTED** [review]
Please remove all the comments from your code, convert syntax to ES6 and add typings

**Comment by Adebayor7** [comment]
okay i would do that

File: src/components/routes/PrivateRoutes.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Outlet, Navigate } from "react-router-dom"
 
 const PrivateRoutes = () => {
-    let auth = {"token": false}
```
Comment (PhilemonBrain) [inline src/components/routes/PrivateRoutes.tsx]: why did u change this?? @Adebayor7

File: src/pages/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    </section>
+  );
+};
+
+
+// const useStyles = makeStyles({
+//     setcol: {
+//         margin: "50px",
+//         padding: "0",
+//         boxSizing: "border-box"
```
Comment (PhilemonBrain) [inline src/pages/Settings.tsx]: remove the comments please

**Review by PhilemonBrain — CHANGES_REQUESTED** [review]

File: src/pages/Settings.tsx (hunk 2 above)
Comment (Adebayor7) [reply src/pages/Settings.tsx]: Okaay done that

**Pull request closed without merging**